apps/core0/subsys/kvm/manager.go

```
@@ -496,6 +509,40 @@ func (m *kvmManager) getDomainStruct(uuid string) (*Domain, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot parse the domain xml: %v", err)
 	}
+
+	domainMetaData, err := domain.GetMetadata(libvirt.DOMAIN_METADATA_ELEMENT, metadataUri, libvirt.DOMAIN_AFFECT_LIVE)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't get tags metadata for domain with the uuid %s", uuid)
+	}
+
+	var metaData MetaData
+	err = xml.Unmarshal([]byte(domainMetaData), &metaData)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't xml unmarshal tags metadata for domain with the uuid %s", uuid)
```

Same, hiding the original error